src/main/java/ui/ExtractMethodPanel.java

```diff
@@ -174,6 +183,7 @@ public void run(@NotNull ProgressIndicator indicator) {
                     model = new ExtractMethodTableModel(new ArrayList<>(candidates));
                     jTree.setModel(model);
                     scrollPane.setVisible(true);
+                    exportButton.setEnabled(!refactorings.isEmpty());
```

Comment:
If we apply all the suggested refactorings, the list with results will be empty and "Export" button will be available. It's incorrect behavior. You should add listener, see `MoveMethodPanel.enableButtonsOnConditions()`